core/sql/regress/executor/EXPECTED130

```diff
@@ -124,7 +134,16 @@ inserted row13
 
 --- 3 row(s) selected.
 >>
->>
+>>values (filetolob('./myfile.txt'));
+
+*** ERROR[8442] Unable to access ExpLOB interface. Call to ExpLOB returned error LOB_PTR_ERROR(532). Error detail 0.
```

Comment:
No we do have an error for the case where a file does not exist. The problem here is that the filetolob() function cannot be called in a values clause.  Same for function stringtolob(). Even if a file is a valid input file, there is no lob column/storage  that the function needs to work on.